strr-web/components/bcros/form-section/BusinessDetails.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div data-test-id="form-section-business-details">
-    <BcrosFormSection :title="t('common.formLabels.businessDetails')">
+    <BcrosFormSection :title="t('common.formLabels.businessDetails')" class="!ml-7">
```

Comment:
Is there another way to do it without enforcing the margin? I recently refactored and it was looking ok.

Reply:
After discussion, this has been removed.